parastell.py

```diff
@@ -203,19 +204,89 @@ def native_export():
     dir = Path(export['dir'])
     filename = Path(export['h5m_filename'])
 
+    def orient_spline_surfaces(volume_id):
+        """Return the inner and outer surface ids for a given volume id
+        """
+        surfaces = cubit.get_relatives('volume', volume_id, 'surface')
+            
+        splineSurfaces = []
+
+        for surf in surfaces:
+            if cubit.get_surface_type(surf) == 'spline surface':
+                splineSurfaces.append(surf)
+
+        # check if this is the plasma
```

Comment:
In the future, there could be other reasons that there is only one surface - perhaps change this comment